From 016ea3cdac7bbb4fd020cdc6a33df746174bd0dc Mon Sep 17 00:00:00 2001
From: Sebastian Walderich <sebastian@walderich.de>
Date: Sat, 8 Feb 2014 08:58:18 +0100
Subject: [PATCH] Corrected variant type in ChoiceByIndexRenderer

As under GTK the wxDataViewChoiceByIndexRenderer expects that the internal
type is equal to its retrieved value type, the internal type has to be
overwritten, so both match.

Closes #15955
---
 src/gtk/dataview.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/gtk/dataview.cpp b/src/gtk/dataview.cpp
index 30a66dc..1af7a04 100644
--- a/src/gtk/dataview.cpp
+++ b/src/gtk/dataview.cpp
@@ -2730,6 +2730,7 @@ wxDataViewChoiceByIndexRenderer::wxDataViewChoiceByIndexRenderer( const wxArrayS
                               wxDataViewCellMode mode, int alignment ) :
       wxDataViewChoiceRenderer( choices, mode, alignment )
 {
+    m_variantType = "long";
 }
 
 void wxDataViewChoiceByIndexRenderer::GtkOnTextEdited(const char *itempath, const wxString& str)
-- 
1.8.5.4

